### ISSUE TITLE
deactivated casa_case can be reactivated

### ISSUE THREAD

**Issue by Zrrrpy** [rubyforgood/casa#1183]
dependent on #1182

AS an **admin**,
I WANT to be able to reactivate a deactivated `casa_case`,
SO THAT `casa_case` details can easily be restored if a `casa_case` reenters the system and requires services from my `casa_org` again.

### How it works now
Currently, `casa_cases` can only be created or edited. After #1182 is complete, `casa_cases` will also be able to be deactivated by an admin.

### How it should work
Admins should be able to reactivated deactivated `casa_cases`. 

### Acceptance Criteria
- [x] On the Edit CASA Case details page of deactivated `casa_cases`, there should be a button to "Activate CASA Case".
- [ ] `casa_case` displays under under "Not Assigned" in "Manage Cases" on "Editing Volunteer" pages after being reactivated (see screenshot below). 
<img width="1375" alt="Screen Shot 2020-10-24 at 5 40 27 PM" src="https://user-images.githubusercontent.com/62810851/97096376-20ccf680-1620-11eb-92bb-758ac379b132.png">

**Comment by RachelWyatt** [comment]
@Zrrrpy is this a duplicate of #1132? I've got the deactivation and reactivation in progress.

**Comment by RachelWyatt** [comment]
PR #1192 partially handles this by adding a "Reactivate CASA Case" Button. 

However the second bullet point still needs to be addressed. Reactivated cases aren't populating in "Not Assigned" on the dropdown.

**Comment by pollygee** [comment]
@Zrrrpy unless I'm reading this ticket incorrectly, I think this is already working.  I deactivated then reactivated a case and then it is showing up on the assign a case page for volunteers.  Maybe I'm misreading and looking in the wrong place? 
![CASA_Volunteer_Tracking](https://user-images.githubusercontent.com/10904005/106633907-4174e880-654d-11eb-99aa-942ef1b36034.jpg)

**Comment by compwron** [comment]
This is done 
![Screen Shot 2021-04-21 at 7 21 08 PM](https://user-images.githubusercontent.com/578159/115646316-c8c53300-a2d6-11eb-9426-ab95bbe4248b.png)
